components/typeform/sources/new-submission/new-submission.mjs

```diff
@@ -22,51 +25,46 @@ export default {
   description: "Emit new submission",
```

Comment:
Bump version in `sources/new-submission.mjs` from `0.0.4` to `0.0.5` since the source and Typeform's app file were updated?